sql/create.sql

```diff
@@ -0,0 +1,49 @@
+CREATE TABLE USERS
+(
+    uid         INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    name        VARCHAR(64) NOT NULL,
+    gh_id       VARCHAR(16) NOT NULL,
+    gh_token    VARCHAR(256) NOT NULL, -- Encrypted GitHub access token
+    cg_token    VARCHAR(256) -- Encrypted? CodeGarten access token

```

Comment:
What is the purpose for this token? Is it to manage an authentication session? If so, it should be in a different table.

Reply:
This token will be used to access the CodeGarten API. As said previously, this atribute is still being discussed and is subject to change.